pkgs/misc/vim-plugins/update.py

```diff
@@ -532,22 +536,35 @@ def update_plugins(input_file: str, outfile: str, cache: Cache) -> Dict:
 
 def main() -> None:
     args = parse_args()
+    if args.add_plugins and not args.commit:
+        raise Exception("The --add argument requires setting the --commit flag.")
     if args.commit:
         nixpkgs_repo = NixpkgsRepo(args.allow_dirty)
     current_plugins = get_current_plugins()
     cache = Cache(current_plugins)
     redirects = {}
 
     redirects = update_plugins(args.input_file, args.outfile, cache)
-
     rewrite_input(args.input_file, redirects)
 
     if args.commit:
-        nixpkgs_repo.commit("Update", [args.outfile])
+        nixpkgs_repo.commit("vimPlugins: Update", [args.outfile])
         if redirects:
             update_plugins(args.input_file, args.outfile, cache)
             nixpkgs_repo.commit(
-                "Update redirects", [args.outfile, args.input_file, DEPRECATED]
+                "vimPlugins: Update redirects",
```

Comment:
Something more explicit might be nice here, such as `resolve github repository redirects`.